README.md

```diff
@@ -28,7 +28,7 @@ The Fission SDK offers tools for:
 import * as sdk from 'webnative'
 
 // Browser/UMD build
-self.webnative
+const sdk = self.webnative
```

Comment:
Should we call this `webnative` instead of `sdk`? I use sdk internally cause of brevity, but it would probably be good to have people thinking of this as "webnative" not "the webnative sdk"

Reply:
Yeah, good question. I agree 👍 Just `wn` instead? 🤔

Reply:
I like it :+1: